packages/editor-preview/src/browser/editor-preview-contribution.ts

```diff
@@ -0,0 +1,72 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { ApplicationShell, KeybindingContribution, KeybindingRegistry, SHELL_TABBAR_CONTEXT_MENU, Widget } from '@theia/core/lib/browser';
+import { Command, CommandContribution, CommandRegistry, MenuContribution, MenuModelRegistry } from '@theia/core/lib/common';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { EditorPreviewWidget } from './editor-preview-widget';
+
+export namespace EditorPreviewCommands {
+    export const PIN_PREVIEW_COMMAND: Command = {
+        id: 'workbench.action.keepEditor',
+        category: 'View',
+        label: 'Keep Editor',
```

Comment:
minor: in VSCode, this label that appears in the context menu is 'Keep Open'

Reply:
> minor: in VSCode, this label that appears in the context menu is 'Keep Open'

Then we would need to update the label at: https://github.com/eclipse-theia/theia/pull/9518/files#diff-14f606c80287893ed697301e733999e7682fe47f8680bb99d05feeb40f446d16R64.

Reply:
I've modified the label below.